### PR TITLE
chore(docs): Fix links to Quick Start

### PR DESCRIPTION
**Pull request by athul**
## Description

Basically Fixes about 5 Quick-Start Link URL bugs. One was in a blogpost(didn't edit that one though)

## Related Issues

PS. Sorry for the 5 Commits :sweat_smile:

### CODE REVIEW

File: docs/docs/sass.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ Sass will compile `.sass` and `.scss` files to `.css` files for you, so you can
 
 ## Installing and Configuring Sass
 
-This guide assumes that you have a Gatsby project set up. If you need to set up a project, head to the [**Quick Start guide**](/docs), then come back.
+This guide assumes that you have a Gatsby project set up. If you need to set up a project, head to the [**Quick Start guide**](/docs/quick-start), then come back.
```
Comment (sidharthachatterjee) [inline docs/docs/sass.md]:
```suggestion
This guide assumes that you have a Gatsby project set up. If you need to set up a project, head to the [**Quick Start guide**](/docs/quick-start/), then come back.
```

File: docs/docs/sourcing-from-drupal.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ It only takes a few steps to use Gatsby with Drupal as a headless CMS (also know
 
 ### Quick start
 
-This guide assumes that you have a Gatsby project set up. If you need to set up a project, head to the [Quick Start guide](/docs), then come back.
+This guide assumes that you have a Gatsby project set up. If you need to set up a project, head to the [Quick Start guide](/docs/quick-start), then come back.
```
Comment (sidharthachatterjee) [inline docs/docs/sourcing-from-drupal.md]:
```suggestion
This guide assumes that you have a Gatsby project set up. If you need to set up a project, head to the [Quick Start guide](/docs/quick-start/), then come back.
```

File: docs/docs/sourcing-from-the-filesystem.md [hunk 3]
```diff
@@ -6,7 +6,7 @@ This guide will walk you through sourcing data from the filesystem.
 
 ## Setup
 
-This guide assumes that you have a Gatsby project set up. If you need to set up a project, please reference the [Quick Start Guide](https://github.com/gatsbyjs/gatsby/tree/master/docs).
+This guide assumes that you have a Gatsby project set up. If you need to set up a project, please reference the [Quick Start Guide](/docs/quick-start).
```
Comment (sidharthachatterjee) [inline docs/docs/sourcing-from-the-filesystem.md]:
```suggestion
This guide assumes that you have a Gatsby project set up. If you need to set up a project, please reference the [Quick Start Guide](/docs/quick-start/).
```

File: docs/tutorial/part-eight/index.md [hunk 4]
```diff
@@ -234,7 +234,7 @@ Lighthouse is a great tool for site improvements and learning -- Continue lookin
 
 ### Official Documentation
 
-- [Official Documentation](https://www.gatsbyjs.org/docs/): View our Official Documentation for _[Quick Start](https://www.gatsbyjs.org/docs/)_, _[Detailed Guides](https://www.gatsbyjs.org/docs/preparing-your-environment/)_, _[API References](https://www.gatsbyjs.org/docs/gatsby-link/)_, and much more.
+- [Official Documentation](https://www.gatsbyjs.org/docs/): View our Official Documentation for _[Quick Start](https://www.gatsbyjs.org/docs/quick-start)_, _[Detailed Guides](https://www.gatsbyjs.org/docs/preparing-your-environment/)_, _[API References](https://www.gatsbyjs.org/docs/gatsby-link/)_, and much more.
```
Comment (sidharthachatterjee) [inline docs/tutorial/part-eight/index.md]:
```suggestion
- [Official Documentation](https://www.gatsbyjs.org/docs/): View our Official Documentation for _[Quick Start](https://www.gatsbyjs.org/docs/quick-start/)_, _[Detailed Guides](https://www.gatsbyjs.org/docs/preparing-your-environment/)_, _[API References](https://www.gatsbyjs.org/docs/gatsby-link/)_, and much more.
```

**Comment by athul** [comment]
Thanks for the Suggestions @sidharthachatterjee The Whole Gatsby Team is :sparkling_heart:

**Pull request merged**